src/main/java/uk/gov/hmcts/reform/sscs/ccd/CcdClient.java

```diff
@@ -37,7 +44,36 @@ public CcdClient(@Autowired CoreCaseDataApi coreCaseDataApi, @Autowired IdamServ
         );
 
         return caseDetailsList.stream()
-                .map(CcdUtil::getCaseDetails)
+                .map(sscsCcdConvertService::getCaseDetails)
                 .collect(toList());
     }
+
+    public SscsCaseDetails createCase(CcdRequestDetails ccdRequestDetails, SscsCaseData caseData) {
```

Comment:
Can we mark this with the `@retryable` annotation please? Example here: https://github.com/hmcts/tribunals-case-api/blob/master/src/main/java/uk/gov/hmcts/sscs/service/ccd/CreateCoreCaseDataService.java#L28

And maybe reuse some of the logging found in this method if possible? This is because we frequently see problems around these lines and helps when trying to debug where the issue is occuring